src/Comment/CommentContainer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sabberworm\CSS\Comment;
+
+/**
+ * Provides a standard reusable implementation of `Commentable`.
+ *
+ * @internal
+ */
+trait CommentContainer
```

Comment:
I haven't had time for a full review yet (customer project, and gave a workshop today) and will do one later.

For the trait, we can add `@phpstan-require-implements Commentable`: https://phpstan.org/writing-php-code/phpdocs-basics#enforcing-implementing-an-interface-for-traits

Reply:
Done.